pkg/image/airgap.go

```diff
@@ -162,7 +174,20 @@ func TagAndPushImagesFromBundle(airgapBundle string, options imagetypes.PushImag
 		if err := tarGz.Unarchive(airgapBundle, extractedBundle); err != nil {
 			return errors.Wrap(err, "falied to unarchive airgap bundle")
 		}
-		return PushImagesFromTempRegistry(extractedBundle, airgap.Spec.SavedImages, options)
+		if err := PushImagesFromTempRegistry(extractedBundle, airgap.Spec.SavedImages, options); err != nil {
+			return errors.Wrap(err, "failed to push images from temp registry")
+		}
+
+		pushEmbeddedArtifactsOpts := imagetypes.PushEmbeddedArtifactsOptions{
+			Registry:   options.Registry,
+			Tag:        airgap.Spec.VersionLabel,
```

Comment:
Using the version label as the tag for these artifacts. Could alternatively use `<channelID>-<channelSequence>` if we need it to be unique as a vendor could theoretically re-use a version label.

Reply:
`<channelID>-<channelSequence>-<versionLabel>` and then sanitize and truncate?